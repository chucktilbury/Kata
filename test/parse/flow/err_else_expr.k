;-------------------------------------------------------------------------------
; expression after else is not allowed
;-------------------------------------------------------------------------------

func name()() {

    if(comp.name == 0) {
        bulart = 0
    }
    else(1 != 2) {
        goop()()
    }
}