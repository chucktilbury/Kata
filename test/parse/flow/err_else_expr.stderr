Syntax: /home/chuck/Src/Kata/test/parse/flow/err_else_expr.k: 10: 10: expected a function body but got (
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_else_expr.k: 10: 11: expected function body element but got literal signed
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_else_expr.k: 11: 13: expected function body but got symbol
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_else_expr.k: 11: 15: expected module item or end of file but got )
Warning: /home/chuck/Src/Kata/test/parse/flow/err_else_expr.k: 11: 17: module is empty
